profile.md

```diff
@@ -234,7 +234,14 @@ The following properties are defined for usage within a Kubernetes deployment:
   Version: 2.14
 
   The unique identifier for the Kubernetes cluster from which the request
-  was sent. This property MUST be a non-empty string serialized as follows:
+  was sent. This property can be useful when a Service Broker tries to
+  call back to a Platform that is managing multiple Kubernetes clusters.
+  Typically, this value will be static for the lifetime of the cluster,
+  however, that is a Platform implementation choice. Brokers ought to take
+  care that while atypical, it is possible for the value to change over
+  time.
```

Comment:
> it is possible for the value to change over  time.

What should happen in this case? What does it mean for the cluster id to change and what happens to the peers that knew what that old id was? (and maybe tied to resources in that cluster?)

Reply:
I'm inclined to leave that as an impl detail because I'm afraid that anything I say might be interpreted as telling people what they MUST do, even if I don't use MUST in the text. For example, if I was to try to say something it might be something like:
- - -
When communicating back to the Platform, Service Brokers will use the clusterID of the most recent message sent by the Platform related to the resource in the Service Broker being accessed.
- - -
But that implies they shouldn't cache it, and while I agree that would be good not to, I'm not sure its our place to say so.  But if you can think of other text I'm not against considering it.